### PR TITLE
Code and prose removed

### LINKED ISSUE #48: Remove "Code" & "Prose". Only have "Data", "Entity", & "Logs".

### PR DESCRIPTION
**Pull request by cbrzn**
I'm setting test_passes as base because I think It'd be good to merge and check if test passes before merging to master

I tested locally (`npm run start`) and it worked

closes #48

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,6 +35,7 @@ ExampleDAOView() => (
 )
 ```
 
+(?)
```
Comment (cbrzn) [inline README.md:38]: This documentation should be changed, my understanding is that we are going to access to method through `data` object

File: README.md [hunk 2]
```diff
@@ -109,6 +110,7 @@ The below example will:
   )}
   </DAO.Data>
   <Members>
+    (?)
```
Comment (cbrzn) [inline README.md:113]: Same thought as previous comment

File: README.md [hunk 3]
```diff
@@ -140,11 +142,10 @@ For Example: The component [**`<DAO address="0x...">`**](./src/components/DAO.ts
...
 Note: ComponentLists break this rule, but will be covered further down.  
 
-## Component Contexts (Data, Code, Prose)  
-Each component has 3 core contexts:  
+## Component Contexts (Data)  
+Each component has one core context:  
 * Data - GraphQL Schema  
-* Code - Abstracted Protocol Functionality  
-* Prose - TBD  
+* NOT Code - Abstracted Protocol Functionality (?)
```
Comment (cbrzn) [inline README.md:148]: The abstracted protocol functionality will now come from data I think? Just want to make sure to update the readme sir! @dOrgJelli 

Like I said before, please correct me if i'm wrong

**Comment by dOrgJelli** [comment]
Closing and opening up a PR into master

**Pull request closed without merging**